app/code/core/Mage/XmlConnect/Block/Checkout/Payment/Method/List.php

```diff
@@ -229,8 +229,8 @@ public function addCustomerBalanceToXmlObj(Mage_XmlConnect_Model_Simplexml_Eleme
                     'post_name' => 'payment[use_customer_balance]',
                     'code'      => 1,
                     'label'     => $this->__('Use Store Credit (%s available)', $balance),
-                    'is_cover_a_quote' => intval($customerBalanceBlock->isFullyPaidAfterApplication()),
-                    'selected'  => intval($customerBalanceBlock->isCustomerBalanceUsed())
+                    'is_cover_a_quote' => (int)$customerBalanceBlock->isFullyPaidAfterApplication(),
+                    'selected'  => (int)($customerBalanceBlock->isCustomerBalanceUsed()
```

Comment:
@Flyingmana can't understand why you approved the pull request since there is obvious error on this line! Travis CI pass failed for all PHP versions

Reply:
because I missed it, and I do check travisCI not always on a review, only always on a merge

Reply:
ok, consider checking travisCI every time :) It looked scary to me that fatal syntax error is approved.